frontend/src/components/auth/LoginGuard.vue

```diff
@@ -1,16 +1,23 @@
 <template>
-    <RouterView v-if="isLogged" />
-    <LoginForm v-else />
+    <div>
+        <Sidenav v-if="isLogged" />
```

Comment:
this type of template breaks single responsibility, you need to define separate layout that will handle router view and rest parts and use it here or in route settings